README.md

```diff
@@ -18,7 +21,14 @@ No. This is a joke.
 
 ### Y U HATE RUST SO MUCH?
 
-I don't, actually. I believe that those who spend their time asking people to rewrite their projects are probably not themselves active Rust developers, as those active devs are probably busy [writing memory-safe code](https://trac.torproject.org/projects/tor/ticket/11331).
```

Comment:
HTTPS links have `//` which prevents the `()` from being paired, causing errors at the lexing stage. They have been migrated to footnote style which does not wrap URLs inside brackets.